ocfweb/docs/markdown_based.py

```diff
@@ -56,5 +64,6 @@ def get_markdown_docs():
         yield Document(
             name='/' + name,
             title=meta['title'],
-            render=partial(render_markdown_doc, meta, text)
+            render=partial(render_markdown_doc, meta, text),
+            path=path,
```

Comment:
I think a better pattern is probably to add `path` as an argument to the `render_markdown_doc` function (e.g. as the first argument), then add it to the partial call. You don't need to modify `Document` at all this way.